core/src/test/java/org/jdbi/v3/core/internal/ReflectionArrayIteratorTest.java

```diff
@@ -11,35 +11,31 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jdbi.v3.core.util;
+package org.jdbi.v3.core.internal;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 import java.util.Iterator;
 
 import org.junit.Test;
 
-public class ReflectionArrayIteratorTest
-{
+public class ReflectionArrayIteratorTest {
     @Test
-    public void testIntArray()
-    {
+    public void testIntArray() {
         final Iterator<Object> it = new ReflectionArrayIterator(new int[]{1, 2, 3});
 
         assertThat(it).containsExactly(1, 2, 3);
     }
 
     @Test
-    public void testEmptyArray()
-    {
+    public void testEmptyArray() {
```

Comment:
heh, I think I code formatted the entire class a few times when I was only targeting a few lines.